migrations/20201124134824-modify_users_add_username.js

```diff
@@ -3,7 +3,7 @@
 module.exports = {
   up: async (queryInterface, _Sequelize) => {
     await queryInterface.sequelize.query(
-      "ALTER TABLE Users ADD COLUMN username VARCHAR(50) NOT NULL UNIQUE DEFAULT(email) AFTER id;"
+      "ALTER TABLE Users ADD COLUMN username VARCHAR(50) NOT NULL UNIQUE;"
```

Comment:
Không sửa file này ở đây e ơi. Không sửa file migration. Nó chỉ để migration chứ nó k phải model

Reply:
Để chị giải thích ha. MIgration là một file để change database, chỉ apply đúng 1 lần duy nhất, những lần sau khi em chạy run migrate nó cũng sẽ ko apply lại file này nếu đã apply rồi.
Cho nên dòng này chỉ có nghĩa, add new column và username = email (để thoả mãn đk unique và not null)
Còn trong model em thì username k bị default là email. Nếu cần giải thích thêm em cứ nhắn qua cho chị nhé.